src/osha/oira/upgrade/alembic_statistics/env.py

```diff
@@ -1,5 +1,5 @@
 from alembic import context
-from osha.oira.statistics.model import Base
+from oira.statistics.deployment.model import Base
```

Comment:
It surprises me to see the alembic upgrades for the `oira.statistic.deployment` package in this package.

Reply:
> It shouldn't surprise you if you've read [syslabcom/scrum#2348](https://github.com/syslabcom/scrum/issues/2348). 

I read it once again and me still surprised :smile:.
I will setup a call to clarify this.

> The point is to correct the previous approach of doing alembic upgrades here in osha.oira and start doing them in oira.statistics.deployment instead. 

This is exactly why I am thinking that this file should not be here anymore.

> Maybe a good further step would be to remove the old upgrades that operate on the statistics databases from osha.oira.

Sure, I would not bother about that.